packages/lab/src/__tests__/stepped-tracker/SteppedTracker.cy.tsx

```diff
@@ -0,0 +1,208 @@
+import { Step, SteppedTracker } from "@salt-ds/lab";
+import { composeStories } from "@storybook/react";
+
+import * as steppedTrackerStories from "@stories/stepped-tracker/stepped-tracker.stories";
+
+const { VerticalDepth1, VerticalDepth2 } = composeStories(
+  steppedTrackerStories,
+);
+
+describe("<SteppedTracker />", () => {
+  it("should expand/collapse when trigger is clicked (depth 1)", () => {
+    cy.mount(<VerticalDepth1 />);
+
+    cy.get("#step-1").should("be.visible");
+    cy.get("#step-1-1").should("not.be.visible");
+
+    cy.get("#step-1-expand-trigger").should(
+      "have.attr",
+      "aria-expanded",
+      "false",
+    );
+
+    cy.get("#step-1-expand-trigger").click();
+
+    cy.get("#step-1-expand-trigger").should(
+      "have.attr",
+      "aria-expanded",
+      "true",
+    );
+
+    cy.get("#step-1-1").should("be.visible");
+
+    cy.get("#step-1-expand-trigger").click();
+
+    cy.get("#step-1-expand-trigger").should(
+      "have.attr",
+      "aria-expanded",
+      "false",
+    );
+
+    cy.get("#step-1-1").should("not.be.visible");
+  });
+
+  it("should expand/collapse when trigger is clicked (depth 2)", () => {
+    cy.mount(<VerticalDepth2 />);
+
+    cy.get("#step-2").should("be.visible");
+    cy.get("#step-2-1").should("not.be.visible");
+    cy.get("#step-2-2").should("not.be.visible");
+    cy.get("#step-2-2-1").should("not.be.visible");
+    cy.get("#step-2-2-2").should("not.be.visible");
+    cy.get("#step-2-2-3").should("not.be.visible");
+
+    cy.get("#step-2-expand-trigger").click();
+
+    cy.get("#step-2-1").should("be.visible");
+    cy.get("#step-2-2").should("be.visible");
+
+    cy.get("#step-2-2-expand-trigger").click();
+
+    cy.get("#step-2-2-1").should("be.visible");
+    cy.get("#step-2-2-2").should("be.visible");
+    cy.get("#step-2-2-3").should("be.visible");
+
+    cy.get("#step-2-expand-trigger").click();
+
+    cy.get("#step-2").should("be.visible");
+    cy.get("#step-2-1").should("not.be.visible");
+    cy.get("#step-2-2").should("not.be.visible");
+    cy.get("#step-2-2-1").should("not.be.visible");
+    cy.get("#step-2-2-2").should("not.be.visible");
+    cy.get("#step-2-2-3").should("not.be.visible");
+  });
+
+  it("a11y/aria properties (depth 0)", () => {
+    cy.mount(
+      <SteppedTracker id="stepped-tracker-1" orientation="vertical">
```

Comment:
arguably don't need an "a11y/aria properties " test if we do not use ids and rely on the aria labels to access the dom

Reply:
Internally I do use ids even when id is not passed, relying on useId(defaultId), so this test is more of a check if everything is linked properly for a11y to be in place.